packages-ts/starknet-gauntlet-starkgate/src/commands/bridge/deploy.ts

```diff
@@ -30,7 +30,7 @@ const makeContractInput = async (
   input: UserInput,
   context: ExecutionContext,
 ): Promise<ContractInput> => {
-  const defaultWallet = context.wallet.getAccountPublicKey()
+  const defaultWallet = context.wallet.getAccountAddress()
```

Comment:
Let's fix the naming here:
```suggestion
  const accountAddr = context.wallet.getAccountAddress()
```

We should stop making the code intentionally hard to understand.

Reply:
I'll address the naming in Bridge PR. We inherited the naming with the whole gauntlet implementation.